modules/core/src/iterators/make-stream/make-dom-stream.ts

```diff
@@ -5,9 +5,9 @@ export type MakeStreamOptions = {
   highWaterMark?: number;
 };
 
-/** 
- * Builds a DOM stream from an iterator 
- * This stream is currently used in browsers only, 
+/**
+ * Builds a DOM stream from an iterator
+ * This stream is currently used in browsers only,
```

Comment:
@ibgreen CI was failing due to a linting issue so I pushed this trivial fix

Reply:
OK if it works we can land, cherry-pick to 3.1-release, clean up the browser fields and and try to publish another patch.